### ISSUE TITLE
[CRD] IPv6 support for virtualServerAddress

### ISSUE THREAD

**Issue by tkam8** [F5Networks/k8s-bigip-ctlr#1857]
#### Title
Need to be able to set IPv6 addresses as VIP/virtualServerAddress

#### Description
CIS Version : 2.4.1
Build: f5networks/k8s-bigip-ctlr:latest
BIGIP Version: Big IP 14.1.0.1.0.0.7
AS3 Version: 3.22.1
Agent Mode: AS3
Orchestration: K8S
Orchestration Version: v1.18.6
Pool Mode: Nodeport

#### Actual Problem
The environment is purely IPv6 environment where internal and external networks are IPv6 only. No dual stack.
This means CIS must support setting IPv6 addresses in CRDs.

Today, we get below error when trying to configure:
```
The VirtualServer "git-virtual-server" is invalid: spec.virtualServerAddress: Invalid value: "[fdfb:8dc4:2a69:2e63:0000:0000:0000:0001]`": spec.virtualServerAddress in body should match '^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$'
```
#### Solution Proposed
Enable capability to configure IPv6 addresses for virtualServerAddress in all CRDs that allow setting of VIPs

#### Alternatives
Use AS3 configmaps, however, migration to CRDs is high priority for usability improvement (basic use cases), automation capability purpose

#### Additional context
This capability will also go hand-in-hand with IPv6 supportability with type loadbalancer, F5 IPAM controller.

Other IPv6 related issues/enhancements:
https://github.com/F5Networks/k8s-bigip-ctlr/issues/1572
https://github.com/F5Networks/k8s-bigip-ctlr/issues/1216
https://github.com/F5Networks/f5-ipam-controller/issues/31

**Comment by andreasjacobi** [comment]
Hi,
What is the status on this enhancement? We are about to migrate to a IPv6 native k8s environment and require the CIS to support VirtualServer, TransportServer and Ingress Service but it currently blocks that migration.

**Comment by trinaths** [comment]
@andreasjacobi - This RFE will be addressed in ~ 2.7/2.8
